### PR TITLE
[WIP] - DO NOT MERGE - 8292 Add WPCC engine

### PR DESCRIPTION
**Pull request by willhall88**
**Target Process ticket**
https://moneyadviceservice.tpondemand.com/entity/8292

**Summary**
This PR mounts WPCC. 
`/en/tools/wpcc`

**Checklist**
+ [x] Tests passed.
+ [ ] Rubocop is passing for this PR (or any other lint like JShint).
+ [ ] Code Climate passed for this PR.
+ [ ] Commit history reviewed (clear commit messages).

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1768)

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -8,7 +8,6 @@ GOOGLE_API_CX_CY=xxxx:xxxx
 STATSD_HOST=localhost
 STATSD_PORT=8125
 WEBCHAT_DOMAIN=xxxx.xxxx
-FRONTEND_HTTP_REQUEST_TIMEOUT=xx
```
Comment (Tr4pSt3R) [inline .env-example]: Good point. This had to be removed as this `FRONTEND_HTTP_REQUEST_TIMEOUT=10` should be left as is without making changed.

File: lib/tool_mount_point/wpcc.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
+    CY_ID = 'wpcc'
```
Comment (Tr4pSt3R) [inline lib/tool_mount_point/wpcc.rb]: The English and Welsh IDs can be found in the description for this [story here](https://moneyadviceservice.tpondemand.com/entity/8191).

```
EN: https://www.moneyadviceservice.org.uk/en/tools/workplace-pension-contribution-calculator

CY: https://www.moneyadviceservice.org.uk/cy/tools/cyfrifiannell-cyfraniadau-pensiwn-gweithle
```

Comment (willhall88) [inline lib/tool_mount_point/wpcc.rb]: thank you! 
Will get them updated

**Comment by tomas-stefano** [comment]
Closing the PR in favor of another PR that will be based on the user story of deploying the wpcc to a secret url

**Pull request closed without merging**